pkgs/racket-doc/scribblings/reference/equality.scrbl

```diff
@@ -632,3 +633,170 @@ and only access mutable data when the mode is true:
           (rec (mcell-value self))))])
 ]
 ]
+
+@section{Combining Hash Codes}
+
+@note-lib-only[racket/hash-code]
+
+@defproc[(hash-code-combine [hc exact-integer?] ...) fixnum?]{
+  Combines the @racket[hc]s into a @tech{hash code} that
+  depends on the order of the inputs.
+  Useful for combining the hash codes of different fields in
+  a structure.
+
+  @examples[
+    (require racket/hash-code)
+    (struct ordered-triple (fst snd thd)
+      #:methods gen:equal+hash
+      [(define (equal-proc self other rec)
+         (and (rec (ordered-triple-fst self) (ordered-triple-fst other))
+              (rec (ordered-triple-snd self) (ordered-triple-snd other))
+              (rec (ordered-triple-thd self) (ordered-triple-thd other))))
+       (define (hash-proc self rec)
+         (hash-code-combine (rec (ordered-triple-fst self))
```

Comment:
Since this demonstrates how `hash-code-combine` can be used for `hash-proc`, I think it would be nice to show the best practice, which probably should include the type tag info.

Reply:
Okay, I can add `(eq-hash-code struct:name)` to these examples.

Reply:
I didn't add `(eq-hash-code struct:wrap)` to the `(hash-code-combine (rec (wrap-value self)))` examples because the point of those was to show `hash-code-combine` with only one argument.

Reply:
Does it make sense to add `(code:comment ...)` saying that "in practice, it is advisable to pass `(eq-hash-code struct:wrap)` into `hash-code-combine`"?

Reply:
I added a comment saying this:
```racket
; demonstrates `hash-code-combine` with only one argument
; but it's good to combine `(eq-hash-code struct:wrap)` too
```

Reply:
Thanks!